Математическая модель динамики самолёта

Erziaikina opened the pull request.

Erziaikina committed:
Математическая модель динамики самолёта

Erziaikina committed:
.

Erziaikina committed:
Мат. модель ЛА с моделированием посадки

NoMonsters reviewed (approved):
Хороша работа
```
d=====(￣▽￣*)b
```
Теперь необзодимо сделать отскок при достижении земли.
ПС в следующих ПР описывай, что было доработано.

Erziaikina committed:
Create Model_plane6_new_land_animation.slx

Добавила реакцию демпфера, при посадке. Вроде работает, но не уверенна, что правильно. Буду ещё проверять. Добавила 3D анимацию полёта ЛА, но не разодралась в настройках камеры, чтобы она следовала за самолётом.
